config/environments/development.rb

```diff
@@ -11,14 +11,31 @@
   # Do not eager load code on boot.
   config.eager_load = false
 
+  # Don't use this literally; use your local dev host instead
+  host = 'localhost'
+  config.action_mailer.raise_delivery_errors = true
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.default_url_options = {host: host, port: 3000 }
```

Comment:
sau { và trước } ko có space nha
```suggestion
  config.action_mailer.default_url_options = {host: host, port: 3000}
```